examples/ConvertAndRead/ConvertAndRead.ino

```diff
@@ -19,7 +19,7 @@ void setup(void)
   digitalWrite(9, HIGH);
   
   // Reset devices
-  MCP342x::generalCallReset();
+  adc.generalCallReset();
```

Comment:
Yes, it is necessary. It cannot be compiled (I used platformio not arduino ide) because it is not an [static member function](https://en.cppreference.com/w/cpp/language/static) anymore. 
In the original code, "Wire" is an static member, so you can define them to static member functions. But when you use an changeable "wire"(It is the functions we need), it become an normal functions. I think it is the reason why you remove "static" in  647b2ce7b44c4b01f57ad449c69fee14fff5f930. So I chose to change MCP342x to adc. Thanks.